openmmtools/integrators.py

```diff
@@ -999,31 +999,38 @@ class LangevinSplittingIntegrator(ThermostatedIntegrator):
         - g-BAOAB with solvent-solute splitting, K_r=K_p=2:
             splitting="V0 V1 R R O R R V1 R R O R R V1 V0"
 
+    Attributes
+    ----------
+    _kinetic_energy : str
+        This is 0.5*m*v*v by default, and is the expression used for the kinetic energy
+
     References
     ----------
     [Leimkuhler and Matthews, 2015] Molecular dynamics: with deterministic and stochastic numerical methods, Chapter 7
     """
 
+    _kinetic_energy = "0.5 * m * v * v"
+
     def __init__(self,
                  splitting="V R O R V",
                  temperature=298.0 * simtk.unit.kelvin,
-                 collision_rate=91.0 / simtk.unit.picoseconds,
+                 collision_rate=1.0 / simtk.unit.picoseconds,
                  timestep=1.0 * simtk.unit.femtoseconds,
                  constraint_tolerance=1e-8,
-                 override_splitting_checks=False,
                  measure_shadow_work=False,
                  measure_heat=True,
-                 measure_protocol_work=False
                  ):
         """Create a Langevin integrator with the prescribed operator splitting.
 
         Parameters
         ----------
         splitting : string
-            Sequence of R, V, O (and optionally V{i}) substeps to be executed each timestep.
+            Sequence of R, V, O (and optionally V{i}), and { } substeps to be executed each timestep.
 
             Forces are only used in V-step. Handle multiple force groups by appending the force group index
             to V-steps, e.g. "V0" will only use forces from force group 0. "V" will perform a step using all forces.
+            { will cause metropolization, and must be followed later by a }.
```

Comment:
Maybe note that the portion in between `{` and `}` will be Metropoloized, and must include all steps which generate shadow work?